src/menu.cpp

```diff
@@ -33,6 +36,15 @@ void Menu::extractFileInfo(const std::string &edgesFilename, const std::string &
     if (edgesFilename.contains("Real-world-Graphs"))
         start = dataRepository.getFurthestVertex().getId();
     auto result = graph.nearestInsertionLoop(start);*/
+    auto start = std::chrono::high_resolution_clock::now();
+    graph.initDistanceMatrix();
```

Comment:
This function doesn't exist and so the project wouldn't even build, please test your work before pushing